src/views/CreatePage.vue

```diff
@@ -75,6 +75,9 @@ const stage = ref()
 const transformer = ref()
 const usersId = localStorage.getItem('usersId')
 const canvasId = ref(useRoute().params.canvas_id)
+const router = useRouter()
+
+const saveState = ref('normal')
```

Comment:
他の文字列が入らないように、saveStateの型を定義しておきたいです。

```suggestion
type SaveState = 'normal' | 'loading' | done

const saveState = ref('normal')
```